### PR TITLE
Add missing MediaList subpages

### PR DESCRIPTION
**Pull request by teoli2003**
This is part of openwebdocs/project#152

Although supported by all three major engines, several methods, and properties, of the basic `MediaList` instance were not documented. This PR adds them.

Note that I have added non-live examples only as the result is not visual (and very complex as the stylesheet of the Live Sample gets in the way).

### CODE REVIEW

File: files/en-us/web/api/medialist/deletemedium/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: MediaList.deleteMedium()
+slug: Web/API/MediaList/deleteMedium
+page-type: web-api-instance-method
+browser-compat: api.MediaList.deleteMedium 
+---
+
+{{APIRef("CSSOM")}}
+
+The `deleteMedium()` method of the {{DOMxRef("MediaList")}} interface removed the media typquerye given in parameter from the list.
```
Comment (dawei-wang) [inline files/en-us/web/api/medialist/deletemedium/index.md:10]:
```suggestion
The `deleteMedium()` method of the {{DOMxRef("MediaList")}} interface removed the media type query given in parameter from the list.
```

File: files/en-us/web/api/medialist/index.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ The **`MediaList`** interface represents the media queries of a stylesheet, e.g.
 - {{domxref("MediaList.deleteMedium()")}}
   - : Removes a media query from the `MediaList`.
 - {{domxref("MediaList.item()")}}
-  - : A getter that returns a string representing a media query as text, given the media query's index value inside the `MediaList`.
+  - : A getter that returns a string representing a media query as text, given the media query's index value inside the `MediaList`. This methods can also be called using the bracket (`[]`) syntax.
```
Comment (dawei-wang) [inline files/en-us/web/api/medialist/index.md:34]:
```suggestion
  - : A getter that returns a string representing a media query as text, given the media query's index value inside the `MediaList`. This method can also be called using the bracket (`[]`) syntax.
```

File: files/en-us/web/api/medialist/length/index.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+title: MediaList.length
+slug: Web/API/MediaList/length
+page-type: web-api-instance-property
+
+browser-compat: api.MediaList.length
+---
+
+{{APIRef("CSSOM")}}
+
+The read-only **`length`** property of the {{DOMxRef("MediaList")}} interface returns the number of media queries in the list
```
Comment (dawei-wang) [inline files/en-us/web/api/medialist/length/index.md:11]:
```suggestion
The read-only **`length`** property of the {{DOMxRef("MediaList")}} interface returns the number of media queries in the list.
```

File: files/en-us/web/api/medialist/appendmedium/index.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+---
+title: MediaList.appendMedium()
+slug: Web/API/MediaList/appendMedium
+page-type: web-api-instance-method
+browser-compat: api.MediaList.appendMedium 
+---
+
+{{APIRef("CSSOM")}}
+
+The `appendMedium()` method of the {{DomxRef("MediaList")}} interface add a media query to the list. If the media query is already in the collection, this method does nothing.
```
Comment (dawei-wang) [inline files/en-us/web/api/medialist/appendmedium/index.md:10]:
```suggestion
The `appendMedium()` method of the {{DomxRef("MediaList")}} interface adds a media query to the list. If the media query is already in the collection, this method does nothing.
```

File: files/en-us/web/api/medialist/deletemedium/index.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+### Exceptions
+
+- `NotFoundError` {{domxref("DOMException")}}
+  - : Thrown when the media query to remove is not in the list.
+
+## Examples
+
+The following removes the media query `print` into the
+`MediaList` associated with the first stylesheet applied to the current document.
```
Comment (dawei-wang) [inline files/en-us/web/api/medialist/deletemedium/index.md:35]:
```suggestion
The following removes the media query `print` from the
`MediaList` associated with the first stylesheet applied to the current document.
```

**Review by dawei-wang — COMMENTED** [review]
Various typos

File: files/en-us/web/api/medialist/item/index.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+### Parameters
+
+- `index`
+  - : An integer.
+
+### Return value
+
+A string, or `null` if there is no entry for the index.
+
+If there is no entry for the given index, and using the bracket (`[]`) syntax, `undefined` is returned.
```
Comment (wbamberg) [inline files/en-us/web/api/medialist/item/index.md:30]:
```suggestion
If the bracket (`[]`) syntax is used and there is no entry for the given index, `undefined` is returned.
```

(just a suggestion that IMO reads a little better)

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks @teoli2003 ! A few little comments. Obviously, @dawei-wang 's comments are good too 👍 .

File: files/en-us/web/api/medialist/deletemedium/index.md (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/api/medialist/deletemedium/index.md:10]:
```suggestion
The `deleteMedium()` method of the {{DOMxRef("MediaList")}} interface removes from this `MediaList` the given media query.
```

**Review by wbamberg — APPROVED** [review]
👍 thank you!

**Pull request merged**